src/lime/_backend/html5/HTML5Window.hx

```diff
@@ -668,17 +665,22 @@ class HTML5Window {
 			elementHeight = Browser.window.innerHeight;
 		}
 
-		if (elementWidth != cacheElementWidth || elementHeight != cacheElementHeight) {
+		if (parent.config != null && Reflect.hasField(parent.config, "allowHighDPI") && parent.config.allowHighDPI) {
```

Comment:
Maybe we should store this into a final bool `allowHighDPI` or something not to duplicate this ugly check.